backend/pkg/database/apievent.go

```diff
@@ -35,20 +35,20 @@ const (
 	// NOTE: when changing one of the column names change also the gorm label in APIEvent.
 	methodColumnName               = "method"
 	pathColumnName                 = "path"
-	pathIDColumnName               = "pathId"
-	statusCodeColumnName           = "statusCode"
-	sourceIPColumnName             = "sourceIP"
-	destinationIPColumnName        = "destinationIP"
-	destinationPortColumnName      = "destinationPort"
-	hasSpecDiffColumnName          = "hasSpecDiff" // hasProvidedSpecDiff || hasReconstructedSpecDiff
-	hostSpecNameColumnName         = "hostSpecName"
-	newReconstructedSpecColumnName = "newReconstructedSpec"
-	oldReconstructedSpecColumnName = "oldReconstructedSpec"
-	newProvidedSpecColumnName      = "newProvidedSpec"
-	oldProvidedSpecColumnName      = "oldProvidedSpec"
-	apiInfoIDColumnName            = "apiInfoId"
-	isNonAPIColumnName             = "isNonApi"
-	eventTypeColumnName            = "eventType"
+	pathIDColumnName               = "pathid"
+	statusCodeColumnName           = "statuscode"
+	sourceIPColumnName             = "sourceip"
+	destinationIPColumnName        = "destinationip"
+	destinationPortColumnName      = "destinationport"
+	hasSpecDiffColumnName          = "hasspecdiff" // hasProvidedSpecDiff || hasReconstructedSpecDiff
+	hostSpecNameColumnName         = "hostspecname"
+	newReconstructedSpecColumnName = "newreconstructedspec"
```

Comment:
god. why not new_reconstructed_spec?
https://stackoverflow.com/questions/2878248/postgresql-naming-conventions

Reply:
This is a good question and I wanted to address it in the note and forgot about it.
Main reason is that I didn't wanted to do API changes or quote column name on every call (quote will allow it to stay camelCase).
We have the `sortKey` that is currently defined in the API with the column name
```
  apiEventSortKey:
    name: 'sortKey'
    description: 'Sort key'
    in: 'query'
    type: 'string'
    enum:
      - time
      - method
      - path
      - statusCode
      - sourceIP
      - destinationIP
      - destinationPort
      - hasSpecDiff
      - hostSpecName
    required: true
```

And we are using it here
```
func CreateSortOrder(sortKey string, sortDir *string) string {
	return fmt.Sprintf("%v %v", strings.ToLower(sortKey), strings.ToLower(*sortDir))
}
```

I wanted to avoid
1. Conversions from `camelCase` to `snake_case`
2. Dedicated function to convert the sort key 
3. Changing the swagger definition since it will required UI changes.

If you think it's a must I will implement option number 2 from above and will change all names to `snake_case`

Reply:
It's not a must, but I think that it's better to at least mention this in a todo and why it works currently. I'm really in favour of the conversion function, we can also do it in a separate PR, but maybe better to do it early and not to mess the db in the upgrades later.

Reply:
I've use `snake_case` and implemented a conversion function.
I needed to do some swagger order changes (moved `definitions` to be first in order to use [Reusable Enums](https://swagger.io/docs/specification/2-0/enums/))